src/obi_auth/browser.py

```diff
@@ -0,0 +1,30 @@
+"""Browser handling module."""
+
+import platform
+import webbrowser
+from contextlib import contextmanager
+
+import pyautogui
+
+from obi_auth.exception import ObiAuthError
+
+KEY_INTERVAL = 0.1
```

Comment:
Even from the pyautogui docs I don't get why a delay is needed, maybe a comment can help?

Reply:
Fair enough. In macOS a delay is expected in hot keys combos. If they are clicked simultaneously there are ignored.